OpenSim/Analyses/InverseDynamics.cpp

```diff
@@ -338,8 +337,11 @@ record(const SimTK::State& s)
         for(int i=0; i<nacc; i++) _constraintMatrix(i,j) = (c[i] - _constraintVector[i]);
         f[j] = 0;
     }
+
+    auto coordinates = _modelWorkingCopy->getCoordinatesInMultibodyTreeOrder();
```

Comment:
I do think it introduces a small overhead, but the `InverseDynamic`s Analysis is only being used by RRA (RRA should be using `InverseDynamicsSolver`, but that is another issue) so I am reluctant to optimize for something that should be removed. 

Current PR Appveyor x86 `testRRA`: [11.08s](https://ci.appveyor.com/project/opensim-org/opensim-core/build/4921/job/w95h8obwmfecua0t#L610)
PR #1279 Appveyor x86  `testRRA`: [9.89s](https://ci.appveyor.com/project/opensim-org/opensim-core/build/4918/job/5cbpa88fnqra9cox#L544)

Is this a tolerable hit? If/when we switch to `InverseDynamicsSolver` the issue also goes away.